src/bin/cql-stress-cassandra-stress/settings/cs_args_good_test.in

```diff
@@ -22,3 +22,6 @@ cassandra-stress counter_write no-warmup cl=LOCAL_ONE duration=20m -schema repli
 cassandra-stress counter_write no-warmup cl=THREE duration=20m -schema keyspace=keyspace2
 cassandra-stress counter_write no-warmup cl=ANY duration=20m -schema replication(strategy=NetworkTopologyStrategy) keyspace=keyspace2
 cassandra-stress counter_write no-warmup cl=ALL duration=20m -schema replication(replication_factor=3) keyspace=keyspace2
+cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1) -rate threads=10 -col n=10 size=UNIFORM(1..20)
+cassandra-stress read cl=QUORUM n=10000 -schema replication(key=value) -rate threads=10 -col names=foo,bar,baz
+cassandra-stress read cl=QUORUM n=10000 -schema replication(factor=1, key=value) -rate threads=10 -col size=FIXED(50)
```

Comment:
> c-s option: introduce -col option 
>
> Implemented parsing logic for -col option.
> 
> For reference, see:
> ```
> $ ./cassandra-stress help -col
> Usage: -node [n=?] [size=DIST(?)]
>   OR
> Usage: -node names=? [size=DIST(?)]
> ```

That's great, but what does this option do? The commit message shows some help message, but it looks like it explains how to use the `-node` option...?